tests/util/usedPropTypes.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+const RuleTester = require('eslint').RuleTester;
+const rule = require('../../lib/rules/no-unused-prop-types');
+
+const parsers = require('../helpers/parsers');
+
+const parserOptions = {
+  ecmaVersion: 2018,
+  sourceType: 'module',
+  ecmaFeatures: {
+    jsx: true
+  }
+};
+
+// ------------------------------------------------------------------------------
+// Tests
+// ------------------------------------------------------------------------------
+
+const ruleTester = new RuleTester({parserOptions});
+
+ruleTester.run('no-unused-prop-types', rule, {
```

Comment:
since these are tests using this rule, let's please move them to the test files for that rule.

Reply:
Ok, done. It's in `tests/lib/rules/no-unused-prop-types.js` now